crates/divan_compat/divan_fork/src/divan.rs

```diff
@@ -310,6 +310,32 @@ impl Divan {
 
                 if should_compute_stats {
                     let stats = bench_context.compute_stats();
+                    {
+                        let name = bench_entry.display_name().to_string();
+                        let file = bench_entry.meta().location.file;
+                        let module_path = bench_entry
+                            .meta()
+                            .module_path_components()
+                            .collect::<Vec<_>>()
+                            .join("::");
+                        let uri = format!("{file}::{module_path}::{name}");
+                        let iter_per_round = bench_context.samples.sample_size;
+                        let times_ns: Vec<_> = bench_context
+                            .samples
+                            .time_samples
+                            .iter()
+                            .map(|s| s.duration.picos / 1_000)
+                            .collect();
+                        let max_time_ns = options.max_time.map(|t| t.as_nanos());
+                        ::codspeed::walltime::collect_raw_walltime_results(
```

Comment:
Could be worth it to add a mention that the env var `CODSPEED_CARGO_WORKSPACE_ROOT` is expected to be set when callin the function in the function's docstring (should have caught that in my previous review)